elements/lazy-iframe/lazy-iframe.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { registerReactElement } from 'bulbs-elements/register';
+import { loadOnDemand } from 'bulbs-elements/util';
+
+registerReactElement(
+  'lazy-iframe',
+  loadOnDemand((props) => (
+    <iframe
+      {...props}
+      style={{
+        width: '100%',
+        height: '100%',
+      }}
+    />
+  ))
+);
```

Comment:
@daytonn @kand @spra85 

`<lazy-iframe>` will be first used in the 1996-week coverage to avoid loading all the youtube embed iframes until they are closer to the viewport.